packages/@aws-cdk/aws-sagemaker/lib/scalable-instance-count.ts

```diff
@@ -0,0 +1,66 @@
+import * as appscaling from '@aws-cdk/aws-applicationautoscaling';
+import { Construct } from 'constructs';
+
+/**
+ * The properties of a scalable attribute representing task count.
+ */
+export interface ScalableInstanceCountProps extends appscaling.BaseScalableAttributeProps {
+}
+
+/**
+ * A scalable sagemaker endpoint attribute
+ */
+export class ScalableInstanceCount extends appscaling.BaseScalableAttribute {
```

Comment:
This should probably be its own PR that highlights the feature, like `feat(sagemaker): autoscaling endpoints`. If it's not hard to do, I support splitting it up. But given that I've already reviewed, I'm ok with pushing forward as is too.

Reply:
> But given that I've already reviewed, I'm ok with pushing forward as is too.

Since you've already reviewed these pieces, I'd prefer to keep them as part of this PR (it saves me a few cycles on excising the feature and regenerating integ test snapshots). As part of subsequent revisions on this PR though, if you begin to feel more strongly that auto-scaling should be split apart, I can certainly do it.